service/src/main/java/io/camunda/service/search/filter/FlownodeInstanceFilter.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service.search.filter;
+
+import io.camunda.util.ObjectBuilder;
+
+public record FlownodeInstanceFilter() implements FilterBase {
+
+  public static final class Builder implements ObjectBuilder<FlownodeInstanceFilter> {
+
+    @Override
+    public FlownodeInstanceFilter build() {
```

Comment:
Just to confirm, for this endoint we have no longer the needs of filters.

Is the case that tenantId would be a potential filter here? Also key?